charts/kubeflow-trainer/crds/trainer.kubeflow.org_trainjobs.yaml

```diff
@@ -53,191 +53,16 @@ spec:
                   Annotations to apply for the derivative JobSet and Jobs.
                   They will be merged with the TrainingRuntime values.
                 type: object
-              datasetConfig:
-                description: Configuration of the training dataset.
```

Comment:
Do we have any script to auto-generate CRDs in the Helm Charts ?

Reply:
I've open #2703 that does the copying of the CRDs anytime they are generated.